tools/read-version

```diff
@@ -65,7 +65,13 @@ output_json = '--json' in sys.argv
 src_version = ci_version.version_string()
 version_long = None
 
-if is_gitdir(_tdir) and which("git"):
+# If we're performing CI for a new release branch (which our tooling creates
+# with an "upstream/" prefix), then we don't want to enforce strict version
+# matching because we know it will fail.
```

Comment:
if you ignore an error, how do you know that the error wasn't valid?

how will you know that the actually released version will work?  I guess you test also on landed branches?

Reply:
Travis runs against each commit in master. Once this change lands, CI will run again and `TRAVIS_PULL_REQUEST_BRANCH` will not be set, so we will revert to the existing logic. That will fail until we push an appropriate tag.

So this doesn't _gate_ on having a working version, but we will get angry emails about it if we break it (and I'll be updating the process to ensure that we do it as a matter of course).